application/frontend/controllers/UserController.php

```diff
@@ -4,17 +4,49 @@
 use common\models\User;
 use frontend\components\BaseRestController;
 use Yii;
-use yii\web\NotFoundHttpException;
 
 class UserController extends BaseRestController
 {
+    public function actionIndex() // GET /user
+    {
+        return User::find()->all();
+    }
+
+    public function actionView($employeeId) // GET /user/abc123
+    {
+        $user = User::findOne(['employee_id' => $employeeId]);
+
+        if (isset($user)) {
+            return $user;
+        }
+
+        Yii::$app->response->statusCode = 204;
+    }
+
     public function actionCreate(): User
     {
-        $existingUser = User::findOne([
-            'employee_id' => Yii::$app->request->getBodyParam('employee_id')
-        ]);
+        $user = new User();
+
+        $user->scenario = User::SCENARIO_NEW_USER;
+
+        $user->attributes = Yii::$app->request->getBodyParams();
+
+        parent::save($user);
```

Comment:
It looks like `BaseRestController->save(...)` isn't a static method, so I think you can call it as `$this->save($user)`. You could also make it static, I suppose, which I assume would be fine.